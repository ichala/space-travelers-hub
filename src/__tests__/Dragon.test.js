import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../Redux/configureStore';
import DragonPage from '../Components/Dragon/Dragon';

describe('testing the app', () => {
  it('testing DragonPage UI', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <DragonPage />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
