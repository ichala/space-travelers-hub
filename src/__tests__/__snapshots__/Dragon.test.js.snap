// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the app testing DragonPage UI 1`] = `<div />`;
